internal/pass/pairing/handlers.go

```diff
@@ -35,7 +36,7 @@ func ExtensionConfigureHandler(pairingApp *Pairing) gin.HandlerFunc {
 
 func ExtensionWaitForConnWSHandler(pairingApp *Pairing) gin.HandlerFunc {
 	return func(gCtx *gin.Context) {
-		token, err := tokenFromWSProtocol(gCtx.Request)
+		token, err := connection.TokenFromWSProtocol(gCtx.Request)
 		if err != nil {
 			logging.Errorf("Failed to get token from request: %v", err)
 			gCtx.Status(http.StatusForbidden)
```

Comment:
maybe this also should be `http.StatusUnauthorized`

Reply:
done